### PR TITLE
Add an example license_file entry.

### PR DESCRIPTION
**Pull request by tylere**
Add an example of adding a license file entry, to encourage new recipes to include it.

### CODE REVIEW

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -42,6 +42,7 @@ about:
   # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
   # See https://opensource.org/licenses/alphabetical
```
Comment (jakirkham) [inline recipes/example/meta.yaml]: Maybe add a comment noting that it is strongly encouraged (even if the license doesn't require it) to include the license file.

**Comment by tylere** [comment]
Added a comment re: the license_file entry.

File: recipes/example/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/example/meta.yaml]: Done below.

**Comment by jakirkham** [comment]
Awesome. Thanks. @conda-forge/core, thoughts?

**Comment by mwcraig** [comment]
Looks good, merging. Thanks, @tylere!

**Pull request merged**